src/ripple/consensus/Validations.h

```diff
@@ -192,6 +222,7 @@ class Validations
         decay_result_t<decltype (&Validation::ledgerID)(Validation)>;
     using NodeKey = decay_result_t<decltype (&Validation::key)(Validation)>;
     using NodeID = decay_result_t<decltype (&Validation::nodeID)(Validation)>;
```

Comment:
I'm completely fine with the `decay_result_t` style.  If you want to use the explicit `using` style, consider
```
using NodeID = ripple::NodeID;
```
I believe you can also use
```
using NodeID = NodeID;
```
but you'll leave your readers a bit puzzled about what you're up to...